how-to-guides/sequential-pipelines/scripts/model_validation.py

```diff
@@ -0,0 +1,130 @@
+import neptune.new as neptune
+import neptune.new.integrations.sklearn as npt_utils
+from neptune.new.exceptions import NeptuneException
+from neptune.new.types import File
+from sklearn.metrics import ConfusionMatrixDisplay, classification_report
+from utils import *
+
+run = neptune.init_run(
+    monitoring_namespace="monitoring/validation",
+)
+
+model_name = "pickled_model"
+dataset_name = "features"
+nrows = 1000
+
+# (Neptune) Get dataset features from preprocessing stage
+run[f"preprocessing/dataset/{dataset_name}"].download()
+
+# (Neptune) Get latest model from training stage
+model_key = "PIPELINES"
+project_key = run["sys/id"].fetch().split("-")[0]
+
+try:
+    model = neptune.init_model(
+        with_id=f"{project_key}-{model_key}",  # Your model ID here
+    )
+    model_versions_table = model.fetch_model_versions_table().to_pandas()
+    latest_model_version_id = model_versions_table["sys/id"].tolist()[0]
+
+except NeptuneException:
```

Comment:
```suggestion
except ModelVersionNotFound:
```

Reply:
I see your point, but this exception is not documented.

Only NeptuneException, which is the parent and works for the majority of cases.

https://docs.neptune.ai/help/setting_up_error_handling_function/